README.md

```diff
@@ -739,7 +776,8 @@ async (request: Request) => {
 
 ### inputFromFormData
 
-Extracts a structured object from a `FormData`:
+`inputFromFormData` extracts values from a `FormData` object into an object literal:
```

Comment:
Let me know if the `object literal` assumption is incorrect, as all the input resolvers have the same return type as [importFromSearch](https://github.com/seasonedcc/domain-functions/blob/ba588c1d2ba054b7dacbcb41f6aae0a86a7873d1/src/input-resolvers.ts#L3)

Reply:
Not sure if `plain object` or some other name might be better

Reply:
I'm not sure about the correctness but I like to think of it as structured objects. There's the Js method structuredClone that implies that it has deep properties. I like to highlight the depth as it is a common request from Remix newcomers learning to deal with FormData, so somehow it'd be cool to let it clear.